scalr/data_source_scalr_vcs_provider_test.go

```diff
@@ -0,0 +1,47 @@
+package scalr
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccScalrVcsProviderDataSource_basic(t *testing.T) {
+	rInt := GetRandomInteger()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
```

Comment:
@brezitskiy In my opinion there is has to be more use case:
-  at lease each filter 
- different filter combination
- not found resource error 
- more than one resource error